.editorconfig

```diff
@@ -0,0 +1,12 @@
+root = true
+
+[*]
+charset = utf-8
+end_of_line = lf
+indent_size = 2
+indent_style = space
+insert_final_newline = true
+trim_trailing_whitespace = true
+
+[{Makefile,*.go,go.mod}]
```

Comment:
For `*.go` & `go.mod` files, I prefer to let `gofmt` be opinionated and do the work ! Could you remove them from the match please ?

Reply:
done